products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -144,9 +149,32 @@ public protocol ExternalCommand: class {
 
     /// Called to execute the given command.
     ///
+    /// This method is deprecated in favor of the execute(Command, BuildSystemCommandInterface) method.
```

Comment:
If you want, you could add an explicit deprecation annotation:

    @available(*, deprecated, renamed: "execute(_:_:)")

This would also give you an automatic fix-it in IDEs.

Reply:
Nifty, but this would introduce a build warning on line 170, on the workaround to invoke the deprecated command...